spec/features/managing_steps_spec.rb

```diff
@@ -203,12 +203,12 @@ def and_I_see_the_steps_updated_on_the_step_by_step_details_page
   end
 
   def and_I_write_a_change_note
-    fill_in "Description", with: "I've changed this step by step!"
+    fill_in "Internal note", with: "I've changed this step by step!"
```

Comment:
It feels like we're missing tests about how change notes are displayed.

Reply:
Added tests to capture the new internal notes' structure on `then_the_change_note_should_be_saved`. After we check for the success message we're now also checking that the history timeline was updated and contains the newly inserted note.